libs/entitlement/src/main/java/org/elasticsearch/entitlement/bootstrap/EntitlementBootstrap.java

```diff
@@ -36,7 +37,7 @@ public class EntitlementBootstrap {
     public record BootstrapArgs(
         @Nullable Policy serverPolicyPatch,
         Map<String, Policy> pluginPolicies,
-        Function<Class<?>, String> pluginResolver,
+        Function<Class<?>, PolicyManager.ScopeInfo> scopeResolver,
```

Comment:
Not for this PR, but I wonder if we should rename `PolicyInfo` to `PolicyScope`.

Reply:
Oops, I meant `ScopeInfo`

Reply:
NGL, `ScopeInfo` was kind of a placeholder name. I think I like `PolicyScope`.